Sources/PointFree/Episode/Show.swift

```diff
@@ -261,10 +270,14 @@ private let subscribeView = View<(Database.User?, Episode)> { user, episode -> N
     p(
       [`class`([Class.pf.type.body.leading, Class.padding([.mobile: [.top: 2, .bottom: 3]])])],
       [
-        """
-        This episode is for subscribers only. To access it, and all past and future episodes, become a
-        subscriber today!
-        """
+        episode.subscriberOnly
+          ? """
+            This episode is for subscribers only. To access it, and all past and future episodes, become a
+            subscriber today!
+            """
+          : """
+            Subscribe today and unlock all episodes!
```

Comment:
How about: "This episode is free to all users. To get access to all past and future episodes, become a subscriber today!"